internal/handlers/pg/pgdb/query.go

```diff
@@ -246,20 +246,38 @@ func prepareWhereClause(sqlFilters *types.Document) (string, []any) {
 	var p Placeholder
 
 	for k, v := range sqlFilters.Map() {
-		switch k {
-		case "_id":
-			switch v := v.(type) {
-			case string:
-				filters = append(filters, fmt.Sprintf(`((_jsonb->'_id')::jsonb = %s)`, p.Next()))
-				args = append(args, string(must.NotFail(pjson.MarshalSingleValue(v))))
+		// don't pushdown query operators and dot notation yet
+		if len(k) != 0 && (k[0] == '$' || types.NewPathFromString(k).Len() > 1) {
+			continue
```

Comment:
Shall we create an issue and add todo here :hugs:

Reply:
Yeah, I was planning to add further issues but wanted to discuss it as I'm not sure what's currently in scope of our roadmap. Thanks for noticing that we could add TODO here!